code/matt/django/lab01/django_to_do/to_do_app/views.py

```diff
@@ -0,0 +1,58 @@
+# from audioop import reverse
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponseRedirect
+from django.utils import timezone
+from django.urls import reverse
+
+from .models import Priorities, TodoItem
+
+
+def index(request):
+    # return HttpResponse("Hello World")
+    # context = {"message": "Hello!!!!"}
+    to_do_items = TodoItem.objects.filter(created_date__lte=timezone.now()).order_by(
+        "-created_date"
+    )
+
+    to_do_complete = to_do_items.filter(task_complete=True)
+    to_do_incomplete = to_do_items.filter(task_complete=False)
+
+    context = {
+        "to_do_items": to_do_items,
+        "to_do_complete": to_do_complete,
+        "to_do_incomplete": to_do_incomplete,
+    }
+
+    return render(request, "layout/index.html", context)
+    # return render(request, "layout/index.html", context)
+
+
+def create(request):
```

Comment:
Errors if the user attempts to run a GET request on `create` (http://127.0.0.1:8000/create/)

Suggestion: Handle get requests either be returning an error, or returning to index.